serialization/json_ld/examples/spdx_document3.json

```diff
@@ -6,18 +6,18 @@
       "type": "CreationInfo",
       "specVersion": "3.0.0",
       "created": "2022-12-01T00:00:00Z",
-      "createdBy": ["https://spdx.dev/elements/3F26391C#spdx-dev"],
-      "profile": ["core", "software"],
-      "dataLicense": "https://spdx.org/licenses/CC0-1.0"
+      "createdBy": ["https://spdx.dev/elements/3F26391C#spdx-dev"]
     },
     {
       "type": "SpdxDocument",
       "spdxId": "https://some.namespace#spdxdocument159",
       "creationInfo": "_:creationInfo1",
       "name": "Doc 159 - two File elements",
+      "profileConformance": ["core", "software"],
       "element": [
         "https://some.namespace#SPDXRef-Package",
-        "https://some.namespace#File1",
+        "https://some.namespace#file1",
+        "https://some.namespace#file2",
         "https://spdx.dev/elements/3F26391C#spdx-spec-v2.3",
```

Comment:
the creator `"https://spdx.dev/elements/3F26391C#spdx-spec-v2.3",` is listed in `"element"` of the SpdxDocument but is not contained in the file